### ISSUE TITLE
Using Different vlan for LB VIPs

### ISSUE THREAD

**Issue by gitmedur** [redhat-cop/keepalived-operator#47]
Hi,
Sorry for using github for questions, but wondering whether i can use different vlan for loadbalancer service VIP.Using OpenShift 4 on Vmware ESXi. Implemented & reached LB VIPs when worker nodes' IP and LB VIPs are on the same vlan, but couldn't find a way to move LB VIPs to different vlan. Can it be via a verbatim config? Will you please guide me?
Thanks

**Comment by raffaelespazzoli** [comment]
I'm not a vlan nor a VMWare ESXi expert. I'd recommend to explore how to do that config manually with keepalived. If it is possible then you's have to map the discovered keepalived config to this operator. I'd probably be able to help with that.

**Comment by gitmedur** [comment]
Seems an IP is required on the second network card for the LB IP to be assigned..Assigned an IP to second NIC which is in LB VIP segment. Then keepalived assigned the LB VIP IP.. Trying to digest this..Would appreciate any clarifications..

**Comment by raffaelespazzoli** [comment]
this should be possible, as you can see from the CRD:
```yaml
apiVersion: redhatcop.redhat.io/v1alpha1
kind: KeepalivedGroup
metadata:
  name: keepalivedgroup-router
spec:
  image: registry.redhat.io/openshift4/ose-keepalived-ipfailover
  interface: ens3
  nodeSelector:
    node-role.kubernetes.io/loadbalancer: ""
  blacklistRouterIDs:
  - 1
  - 2 
```
one can specify the NIC/interface name. so put in that field the name of your second NIC.

**Comment by ronnessim** [comment]
We tried to create a configuration with different VLANs, but we ended up with the second keepalive group having ports conflicting with the first:

0/6 nodes are available: 6 node(s) didn't have free ports for the requested pod ports.

We suspect that this is caused by the combination to host networking mode and the Prometheus port (9650) which conflicts with the first instance. Is there a way to change the port to something else in the keepaliveGroup definition?
